modules/store/src/store_module.ts

```diff
@@ -159,22 +169,29 @@ export class StoreModule {
       | InjectionToken<ActionReducerMap<any, any>>
       | ActionReducer<any, any>
       | InjectionToken<ActionReducer<any, any>>,
-    config: StoreConfig<any, any> = {}
+    config: StoreConfig<any, any> | InjectionToken<StoreConfig<any, any>> = {}
   ): ModuleWithProviders<StoreFeatureModule> {
     return {
       ngModule: StoreFeatureModule,
       providers: [
+        {
+          provide: FEATURE_NAME,
```

Comment:
Its easier to override `STORE_FEATURES` as a single token instead of having to override `STORE_FEATURES` and `FEATURE_NAME`